velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -719,15 +720,15 @@ HiveDataSink::maybeCreateBucketSortWriter(
       sortCompareFlags_,
       sortPool,
       writerInfo_.back()->nonReclaimableSectionHolder.get(),
-      spillConfig_);
+      spillConfig_,
+      writerInfo_.back()->spillStats.get());
```

Comment:
Is it possible to pass the spill stats from table write operator to hive data sink? thanks!